Dockerfile

```diff
@@ -81,7 +81,7 @@ FROM rust:1.60.0-alpine as base-optimizer
 
 # Being required for gcc linking
```

Comment:
Let's add this comment since `git-fetch-with-cli` can be necessary in cases of public git dependencies too sometimes.

```suggestion
# musl-dev is required for gcc linking.
# Adding Git for usage within projects that use `git-fetch-with-cli = true`.
```